lib/widgets/recent_dm_conversations.dart

```diff
@@ -45,14 +60,28 @@ class _RecentDmConversationsPageState extends State<RecentDmConversationsPage> w
       appBar: AppBar(title: const Text('Direct messages')),
       body: ListView.builder(
         itemCount: sorted.length,
-        itemBuilder: (context, index) => RecentDmConversationsItem(narrow: sorted[index])));
+        itemBuilder: (context, index) {
+          final narrow = sorted[index];
+          return RecentDmConversationsItem(
+            narrow: narrow,
+            // TODO(i18n) formatting, which we might need the locale for.
+            //   For 9000, do we want 9000 or 9,000 or 9k or 9K or
+            //   (as in Figma) 999+?
+            unreadCount: unreadsModel!.countInDmNarrow(narrow),
```

Comment:
In web we give the exact figure, so I think we want the same here. In particular that way you can see it go down as you read things, even if the total is large. (Not super likely for a DM conversation in particular, but makes sense to be consistent.)

Specifically it looks like we say "9000". So we can just do the same here without a TODO. If we later decide to change that UI choice, we can come back to it (and make it locale-sensitive so it's "9.000" in some locales, etc.).

Reply:
bump — no need for a TODO comment for this

Reply:
Ah oops; forgot about this, thanks.